_docs/0.3/usage/e2e-runner.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Usage - E2E Runner
+version: '0.3'
+categories:
+    - '0.3'
+    - usage
+redirect_from: /docs/usage/dataset-comparison
+---
+
+#### Table Of Contents
+- [Table Of Contents](#table-of-contents)
+- [Properties](#properties)
+- [Parameters](#parameters)
+- [Test Definition JSON](#test-definition-json)
+
+#### Properties
+
+All properties were scraped. Only properties that might come in hadny are the ones implemented atop scribe logging library. Check them out on their GitHub [page](https://github.com/outr/scribe)
```

Comment:
The meaning does not seem to be clear to me. Do you mean _scrapped_ as in _shredded_? But if so, this does not seem proper description terms, why not simply "There are no properties to be defined"?

Or do you actually mean _scraped_ from somewhere? (I am not familiar with Scribe) If so, maybe this would deserve some more description on how this is can be used?

Reply:
I am little confused here as well. I though the implicit reference (of scrapped properties) was to previous version of Hermes. 🤔